File: exercicios/R/1131/1131.r
input = file('stdin','r')

grenais = as.integer(0)
g_inter = as.integer(0)
g_gremio = as.integer(0)
emp = as.integer(0)
cont = as.integer(0)
op = as.integer(0)

while(TRUE){

    line = strsplit(readLines(input,n=1), " ")
    g_i = as.integer(line[[1]][1])
    g_g = as.integer(line[[1]][2])

    if(g_i > g_g){
        g_inter = g_inter + 1
    }else if ( g_i < g_g){
        g_gremio = g_gremio + 1
    }else{
        emp = emp + 1
    }

    cont = cont + 1
    
    write("Novo grenal (1-sim 2-nao)",'')
    op = readLines(input,n=1)

    if(op == 2){
        write(sprintf("%d grenais",cont),'')
        write(sprintf("Inter:%d",g_inter),'')
        write(sprintf("Gremio:%d",g_gremio),'')
        write(sprintf("Empates:%d",emp),'')

        if(g_inter > g_gremio){
            write("Inter venceu mais",'')
        }else if(g_inter < g_gremio){
            write("Gremio venceu mais",'')
        }
        break
    }
}

File: exercicios/JavaScript/1131/1131.js
const { Console } = require('console');

var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var count=0, g_gre=0, g_int=0, emp=0;

while(true){
    var line1 =  lines.shift().split(' ');
    var gol_int = parseInt(line1[0]), gol_gre = parseInt(line1[1]);

    if(gol_int > gol_gre){
        g_int += 1;
    }else if( gol_int < gol_gre){
        g_gre += 1;
    }else{
        emp += 1;
    }
    count += 1;

    console.log("Novo grenal (1-sim 2-nao)")
    
    var new_placar = parseInt(lines.shift());

    if(new_placar == 2){
        console.log(count+ " grenais");
        console.log("Inter:"+ g_int);
        console.log("Gremio:"+ g_gre);
        console.log("Empates:"+emp);

        if(g_int > g_gre){
            console.log("Inter venceu mais");
        }else if(g_int < g_gre){
            console.log("Gremio venceu mais");
        }else{
            console.log("Empate");
        }

        break;
    }
}